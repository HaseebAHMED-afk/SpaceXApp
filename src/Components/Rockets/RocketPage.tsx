import React from 'react'
import Rockets from './Rockets'

const RocketPage = () => {
    return (
        <div>
            <Rockets />
        </div>
    )
}

export default RocketPage
