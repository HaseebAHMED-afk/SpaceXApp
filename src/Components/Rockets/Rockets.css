.rocket{
    color: white;
    text-align: center;
    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
}
.rocket-card-deck{
    flex-wrap: wrap;
    display: flex;
    justify-content: center;
}
.rocket-card{
    width: 25%;
    height:auto;
    background-color: white;
    color: black;
    margin: 15px;
}
.rocket-img{
    width: 100%;
    height: 250px;
    object-fit: cover;
}