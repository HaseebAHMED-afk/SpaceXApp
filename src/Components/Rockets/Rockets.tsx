import React from 'react'

const Rockets = () => {
    return (
        <div>
            RocketPage
        </div>
    )
}

export default Rockets
