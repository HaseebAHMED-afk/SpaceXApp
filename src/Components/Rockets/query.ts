import gql from 'graphql-tag'

export const ROCKET_INFO = gql `
query Rockets{
    rockets {
      active
      id
      flickr_images
      rocket_name
    }
  }
`;