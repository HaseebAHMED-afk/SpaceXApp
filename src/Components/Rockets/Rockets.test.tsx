import React from 'react';
import {shallow} from 'enzyme'
import Rockets from './Rockets';

describe('for Rocket Page' , () => {
    let container : any;
    beforeEach(() => container = shallow(<Rockets />))

    it('renders the rocket componenet', () => {
        expect(container).toMatchSnapshot();
    })
    
    
})