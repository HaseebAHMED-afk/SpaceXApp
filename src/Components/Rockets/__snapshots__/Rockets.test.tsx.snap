// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`for Rocket Page renders the rocket componenet 1`] = `
<div
  className="rocket"
>
  <h1>
    Rockets
  </h1>
  <p>
    The spectacular marvel of engineering that allow SpaceX to fullfil its vision
  </p>
  <div
    className="rocket-card-deck"
  />
  <Footer />
</div>
`;
